contrib/pg_tde/src/keyring/keyring_kmip.c

```diff
@@ -175,17 +175,6 @@ get_key_by_name(GenericKeyring *keyring, const char *key_name, KeyringReturnCode
 			return NULL;
 		}
 
-		if (key->data.len > sizeof(key->data.data))
```

Comment:
This removal will potentially cause a buffer overflow later, at line 180

Reply:
Ah, right. Reverted.